apps/meteor/app/models/server/models/Subscriptions.js

```diff
@@ -30,7 +30,6 @@ export class Subscriptions extends Base {
 		this.tryEnsureIndex({ emailNotifications: 1 }, { sparse: 1 });
 		this.tryEnsureIndex({ autoTranslate: 1 }, { sparse: 1 });
 		this.tryEnsureIndex({ autoTranslateLanguage: 1 }, { sparse: 1 });
-		this.tryEnsureIndex({ 'userHighlights.0': 1 }, { sparse: 1 });
```

Comment:
if we decide to go ahead an remove this index and the one `{ rid: 1 }`, a migration removing them will be needed.